src/utils/generateToken.ts

```diff
@@ -0,0 +1,11 @@
+import Jwt from "jsonwebtoken";
+
+const generateToken = (data: any, options: any) => {
+  const token = Jwt.sign({ data }, process.env.JWT_SECRET as string, options);
+  return token;
+};
+const verifyToken = (token: any) => {
+  const obj = Jwt.verify(token, process.env.JWT_SECRET as string);
+  return obj;
+};
+export { generateToken, verifyToken };
```

Comment:
Check and see if this utility function doesn't exist anywhere else in the codebase so that we do not duplicate work.